docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,56 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+When the parent resource is scoped under a resource group, the path for the list calls of its child resources must also be scoped under a resource group
+
+## How to fix
+
+To fix this, ensure that the LIST call for the child resource has a resource group segment.
```

Comment:
To fix this, ensure that the list operation's api path for the nested resource includes all the parent segments.